templates/zerver/help/editing-past-messages.md

```diff
@@ -0,0 +1,36 @@
+## Editing past messages
+
+Note: Editing topic titles is discussed in a separate answer.
+
+Zulip allows you to easily edit your messages after they have been posted.
```

Comment:
I can change that. But wouldn't it be redudandant, since the previous note makes it clear that this section is only about editing the content?